packages/synapse-react-client/src/components/widgets/query-filter/EnumFacetFilter/EnumFacetFilterOption.tsx

```diff
@@ -24,23 +35,44 @@ export function EnumFacetFilterOption(props: EnumFacetFilterOptionProps) {
       }}
       onMouseEnter={onHover}
     >
-      <Checkbox
-        className="EnumFacetFilter__checkbox"
-        onClick={event => event.stopPropagation()}
-        onChange={newValue => {
-          onChange(newValue)
-        }}
-        key={`${id}`}
-        checked={checked}
-        label={label}
-      ></Checkbox>
-      {isDropdown && (
-        <span className="EnumFacetFilter__count">
-          ({count.toLocaleString()})
-        </span>
+      {inputType === 'checkbox' && (
+        <Checkbox
+          className="EnumFacetFilter__checkbox"
+          onClick={event => event.stopPropagation()}
+          onChange={newValue => {
+            onChange(newValue)
+          }}
+          key={`${id}`}
+          checked={checked}
+          label={label}
+        ></Checkbox>
       )}
-      {!isDropdown && (
-        <div className="EnumFacetFilter__count">{count.toLocaleString()}</div>
+      {inputType === 'radio' && (
+        <RadioOption
+          className="EnumFacetFilter__checkbox"
+          value={id || label}
+          onClick={event => event.stopPropagation()}
+          onChange={() => {
+            onChange(!checked)
+          }}
+          key={`${id}`}
+          checked={checked}
+          label={label}
+        />
+      )}
+      {count != null && (
+        <>
+          {isDropdown && (
+            <span className="EnumFacetFilter__count">
+              ({count.toLocaleString()})
+            </span>
+          )}
+          {!isDropdown && (
+            <div className="EnumFacetFilter__count">
+              {count.toLocaleString()}
+            </div>
+          )}
+        </>
```

Comment:
Allow rendering the option without a value for `count`

Reply:
Not sure if this pattern is preferred, or if setting the display value to inline-block if isDropdown would be better.  This is clean, but has some duplication.